test/Image2.rb

```diff
@@ -286,6 +286,10 @@ def test_destroy
       elsif method == 'difference'
         other = Magick::Image.new(20, 20)
         assert_raises(Magick::DestroyedImageError) { @img.difference(other) }
+      elsif method == 'channel_entropy' && IM_VERSION < Gem::Version.new('6.9')
+        assert_raises(NotImplementedError) { @img.channel_entropy }
+      elsif %w[morphology morphology_channel].include?(method)
+        # noop, need to figure out how to test these
```

Comment:
let's not merge which test disabled in a hidden way like this. better to make it super obvious. like, even

```
STDERR.puts "Skipping thing testing woo"
```

Reply:
Okay, I added a `warn` here. It's not going to be very obvious, however, since there is already a *lot* of stuff being output during a test suite run.